src/compas_rhino/geometry/mesh.py

```diff
@@ -45,13 +45,19 @@ def from_selection(cls):
         guid = compas_rhino.select_mesh()
         return cls.from_guid(guid)
 
+    @classmethod
+    def from_geometry(cls, geometry):
+        mesh = cls()
+        mesh.geometry = geometry
+        return mesh
+
     @property
     def vertices(self):
-        return [map(float, vertex) for vertex in compas_rhino.rs.MeshVertices(self.guid)]
+        return [map(float, vertex) for vertex in compas_rhino.rs.MeshVertices(self.geometry)]
```

Comment:
does this work in Rhino?

Reply:
yes

Reply:
i guess it does because of the coercing mechanism in the background?

Reply:
ah, yes see my updated comment